src/concrete_lattice.jl

```diff
@@ -262,3 +282,20 @@ function loadLattice(
 
     return loadLattice(Lattice, fn, group)
 end
+
+
+"""
+    function getname(l::Lattice)
+
+Function for returning the name of the lattice `l`.
+
+# Examples
+
+```julia-REPL
+julia> name(mylattice)
+:mylattice
+```
+"""
+getname(:: Lattice{S, B, U, name}) where {S, B, U, name} = name
+
+export getname
```

Comment:
Should this, up to the implementation, be moved to `abstract_lattice.jl`? (Same for `getname(::Unitcell)`)